src/dataprotection/azext_dataprotection/manual/_params.py

```diff
@@ -83,6 +86,16 @@ def load_arguments(self, _):
         c.argument('protection_status', arg_type=get_enum_type(get_protection_status_values()), nargs='+', help="specify protection status.")
         c.argument('datasource_id', type=str, nargs='+', help="specify datasource id filter to apply.")
 
+    with self.argument_context('dataprotection backup-instance update-msi-permissions') as c:
+        c.argument('operation', arg_type=get_enum_type(get_backup_operation_values()), help="List of possible operations")
+        c.argument('datasource_type', arg_type=get_enum_type(get_datasource_types()), help="Specify the datasource type of the resource to be backed up")
+        c.argument('vault_name', type=str, help="Name of the vault.")
+        c.argument('permissions_scope', arg_type=get_enum_type(get_permission_scope_values()), help="Scope for assigning permissions to the backup vault")
+        c.argument('keyvault_id', type=str, help='ARM id of the key vault')
+        c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
+        c.argument('backup_instance', type=validate_file_or_dict, help='Request body for operation Expected value: '
```

Comment:
Likewise, we can specify  in the help text that this argument is required if the --operation is Backup